kpm/platforms/kubernetes.py

```diff
@@ -28,6 +28,8 @@
     "configmaps": "api/v1/namespaces/{namespace}/configmaps",
     "replicationcontrollers": "api/v1/namespaces/{namespace}/replicationcontrollers",
     "pods": "api/v1/namespaces/{namespace}/pods",
+    "statefulset": "apis/apps/v1beta1/namespaces/{namespace}/statefulsets",
+    "storageclass": "apis/storage.k8s.io/v1beta1/statefulsets",
```

Comment:
Sorry, there is an error here its :
"storageclass": "apis/storage.k8s.io/v1beta1/storageclasses"